### PR TITLE
Remove Codecov token from .travis.yml

### PR DESCRIPTION
**Pull request by l-vo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Master
+* Remove Codecov token from .travis.yml and put it (after regeneration) in Travis repository settings
```
Comment (emri99) [inline CHANGELOG.md]: I don't think it's required to put it in travis repository settings, as indicated in codecov project setting page

```
Note: Token not required for public repositories uploading from Travis, CircleCI or AppVeyor.
````

```suggestion
* Remove Codecov token from .travis.yml
```

Comment (l-vo) [reply CHANGELOG.md]: Hum so let's try it :) Thanks

**Comment by l-vo** [comment]
@emri99 I saw that while looking to the codecov problem. I don't remember if I made that for temporary purpose or because the token didn't grant dangerous permissions.

**Review by emri99 — APPROVED** [review]

**Comment by l-vo** [comment]
@emri99 thank you :)

**Pull request merged**